File: 백준/Bronze/27889. 특별한 학교 이름/특별한 학교 이름.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String name = br.readLine();
        switch (name.charAt(0)) {
            case 'N':
                System.out.println("North London Collegiate School");
                break;
            case 'B':
                System.out.println("Branksome Hall Asia");
                break;
            case 'K':
                System.out.println("Korea International School");
                break;
            case 'S':
                System.out.println("St. Johnsbury Academy");
                break;
        }
    }
}

File: 백준/Bronze/27889. 특별한 학교 이름/특별한 학교 이름.cc
#include <stdio.h>

int main(void) {
	char name[5];
	scanf("%s", name);
	switch (name[0]) {
	case 'N':
		printf("North London Collegiate School\n");
		break;
	case 'B':
		printf("Branksome Hall Asia\n");
		break;
	case 'K':
		printf("Korea International School\n");
		break;
	case 'S':
		printf("St. Johnsbury Academy\n");
		break;
	}
	return 0;
}